app/scripts/metamask-controller.js

```diff
@@ -1582,7 +1582,16 @@ export default class MetamaskController extends EventEmitter {
         updateTransactions: false,
       },
       isMultichainEnabled: process.env.TRANSACTION_MULTICHAIN,
-      isSimulationEnabled: () => false,
+      isSimulationEnabled: () => {
+        let enabled = false;
+        ///: BEGIN:ONLY_INCLUDE_IF(transaction-simulation)
+        // Only support transaction simulation on mainnet.
+        enabled =
```

Comment:
Sorry for the back and forth, as discussed in the DSM, we no longer need to check the supported networks here as that will be handled by within the `TransactionController` using the networks endpoint.

Reply:
Done.